### PR TITLE
Feature/lp107 community viewmodel

### PR DESCRIPTION
**Pull request by Lupillo17**
Added ViewModel functionality of screen Community

### CODE REVIEW

File: feature/community/src/main/java/com/iteneum/community/presentation/CommunityView.kt [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+ * @modifyBy Jose Rivera
+ **/
+@Composable
+fun CommunityView(
+    navigationToAmenities: () -> Unit,
+    navigationToEvents: () -> Unit,
+    navigationToCommunityWall: () -> Unit,
+    navigationToDoItYourSelf: () -> Unit,
+    navigationToServices: () -> Unit,
+    viewModel: CommunityViewModel = hiltViewModel()
```
Comment (melissaquezada94) [inline feature/community/src/main/java/com/iteneum/community/presentation/CommunityView.kt:50]: Please add the viewmodel to the documentation

File: feature/community/src/main/java/com/iteneum/community/presentation/CommunityView.kt [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                LpIconTextCard(modifier = Modifier
+                    .size(width = sizes.extraSize124, height = sizes.extraSize128)
+                    .padding(sizes.smallerSize),
+                    icon = itemCard.icon,
+                    description = stringResource(
+                        id = cardDescriptions[itemCard.cardType]
+                            ?: R.string.default_card_description
+                    ),
+                    onCardClicked = {
+                        when (itemCard.cardType) {
```
Comment (melissaquezada94) [inline feature/community/src/main/java/com/iteneum/community/presentation/CommunityView.kt:96]: Could you use a sealed class instead of map?

**Review by irv205 — APPROVED** [review]

File: feature/community/src/main/java/com/iteneum/community/presentation/CommunityView.kt [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+                            CommunityCardType.Amenities -> {
```
Comment (melissaquezada94) [inline feature/community/src/main/java/com/iteneum/community/presentation/CommunityView.kt:97]: This is an example: 
Screenshot 2023-04-20 at 7 03 35 PM
Screenshot 2023-04-20 at 7 03 50 PM

Comment (Lupillo17) [reply feature/community/src/main/java/com/iteneum/community/presentation/CommunityView.kt:97]: Sure Solving

**Review by melissaquezada94 — APPROVED** [review]

**Review by melissaquezada94 — APPROVED** [review]

**Pull request merged**